plugins/stats_table_procedure.h

```diff
@@ -113,10 +113,11 @@ struct StatsTableProcedure: public Procedure {
 
 struct StatsTableFunctionConfig {
     StatsTableFunctionConfig(const Url & modelFileUrl = Url())
-        : modelFileUrl(modelFileUrl)
+        : injectNoise(false), modelFileUrl(modelFileUrl)
     {
     }
 
+    bool injectNoise;
```

Comment:
We're trying to transition to C++ 11 member initialization (i.e. bool injectNoise = false)